### PR TITLE
[FIX] - Naming conventions and "demo-friendly" code 

### PR DESCRIPTION
**Pull request by julieahlquist**
### User Story

```
As a product owner,
In order to show code to clients,
I want to have the code base to be free of lewd language and unsavory phrases.
```
### Feature description
Link to this [Chore](https://www.pivotaltracker.com/story/show/168107574)

### Description of PR
Cleans up bad language, bad grammar and aims for better consistency of test descriptions. The reason why this work is done is primarily for demo-purposes to not confuse/upset the person who is introduced to the app.

### What did we learn?
Consistency is key.

### CODE REVIEW

File: cypress/integration/universityCanPaySubscription.spec.js [hunk 1]
```diff
@@ -77,15 +77,15 @@ describe("University pay for subscription", () => {
...
+    cy.contains("Payment failed");
   });
 
   it("Payment fails due to invalid inputs", () => {
     cy.route({
       method: "POST",
       url: "http://localhost:3000/api/v0/subscriptions",
       response: "fixture:unsuccessful_subscription_payment_request.json",
-      status: 402
+      status: 200
```
Comment (ghost) [inline cypress/integration/universityCanPaySubscription.spec.js]: This status message does not align with an unsuccessful response.

**Review by ghost — COMMENTED** [review]
Looks good, and left one comment.

**Review by tochman — APPROVED** [review]
LGTM apart from the status change Max commented on.

File: cypress/integration/universityCanPaySubscription.spec.js (hunk 1 above)
Comment (julieahlquist) [reply cypress/integration/universityCanPaySubscription.spec.js]: fixed!

**Pull request merged**